crates/worker/src/partition/services/non_deterministic/idempotent_invoker.rs

```diff
@@ -167,15 +168,16 @@ impl<'a, State: StateReader + Send + Sync> IdempotentInvokerBuiltInService
         self.set_state(&RESPONSE, &response)?;
 
         // Set response timer
-        self.delay_invoke(
+        self.outbox_message(OutboxMessage::ServiceInvocation(ServiceInvocation::new(
             FullInvocationId::generate(self.full_invocation_id.service_id.clone()),
             restate_pb::IDEMPOTENT_INVOKER_INTERNAL_ON_TIMER_METHOD_NAME.to_string(),
             Bytes::new(),
             Source::Service(self.full_invocation_id.clone()),
             None,
-            expiry_time.into(),
-            0,
-        );
+            SpanRelation::None,
+            vec![],
+            Some(expiry_time.into()),
```

Comment:
s/expiry_time/execution_time/

Reply:
in the context of this code, idempotency_expiration_time makes more sense.

Reply:
Yes, you are right. My comment was stupid.